kubernetes_env/benchmark/benchmark.py

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/env python3
+import argparse
+import logging
+import subprocess
+import os
+import sys
+import signal
+import requests
+import time
+import matplotlib.pyplot as plt
+import json
+from pathlib import Path
+
+sys.path.append("..")
```

Comment:
This might break depending on where the file is called. I would use an import relative to FILE_DIR.